spec/koans.js

```diff
@@ -438,10 +440,11 @@ describe('the JavaScript language', () => {
       const obj = createObject();
       obj.addPoint();
 
-      //expect(obj.score()).toEqual();
-      //expect(typeof(obj.points)).toEqual();
+      expect(obj.score()).toEqual(1);
+      expect(typeof(obj.points)).toEqual(`undefined`);
     });
 
+    //ESTA NO NOS SALIÓ
     it('may create objects also with the module pattern', () => {
       function createObject(initialScore) {
```

Comment:
aqui debian de crear un objeto con funciones 

`
      function createObject(initialScore,initalColor) {
        
        return  {
          color:initalColor,
          score:initialScore,
          incrementScoreIn: (points) => {
            this.score = points + initialScore
          },
          points:()=>this.score,
        }
      }

`